patroni/dcs/__init__.py

```diff
@@ -994,9 +1001,6 @@ def __permanent_slots(self) -> Dict[str, Union[Dict[str, Any], Any]]:
                 if name in slots:
                     # If primary reported flush LSN for permanent slots we want to enrich our structure with it
                     value['lsn'] = slots[name]
-                elif self.is_physical_slot(value) and name == leader_name and leader and leader.lsn:
-                    # there is no slot on the leader for itself, use `lsn` from the member key.
-                    value['lsn'] = leader.lsn
```

Comment:
Am I right in thinking that we are now writing all permanent slots to the `status` key in DCS, including the one for the leader, if it has been configured as a permanent slot?